### PR TITLE
Feature detail page

### PR DESCRIPTION
**Pull request by matystroia**

### CODE REVIEW

File: frontend/src/app/course/courseDetails.jsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
```
Comment (teodorvoicencu) [inline frontend/src/app/course/courseDetails.jsx]: add // @flow at the beginning of the component files

File: frontend/src/app/course/courseHeader.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
```
Comment (teodorvoicencu) [inline frontend/src/app/course/courseHeader.js]: @flow here as well

File: frontend/src/app/course/courseHeader.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+                    <h5>10 slides</h5>
+                </div>
+                <div className={'courseExtraItem'}>
+                    <Albums className={'icon'} />
+                    <h5>something else</h5>
+                </div>
+            </Card>
+        </div>
+    );
+};
```
Comment (teodorvoicencu) [inline frontend/src/app/course/courseHeader.js]: This component can be divided into smaller ones

File: frontend/src/app/course/styles/courseDetails.scss [hunk 4]
```diff
@@ -0,0 +1,9 @@
+@import '../../theme/variables';
+
+.courseDetailsCard {
+    margin: $ternary-margin $ternary-margin;
+}
+
+.courseDetails {
+    padding: $ternary-padding $ternary-padding;
```
Comment (teodorvoicencu) [inline frontend/src/app/course/styles/courseDetails.scss]: 👌🏿

File: frontend/src/app/course/styles/courseHeader.scss [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+.courseTeacher:after {
+    content: '\A';
+    width: 7px;
+    height: 7px;
+    border-radius: 50%;
+    background: #ccc;
+    margin-left: 7px;
+    margin-right: 7px;
+    margin-bottom: 3px;
```
Comment (teodorvoicencu) [inline frontend/src/app/course/styles/courseHeader.scss]: Please use rem instead of px

File: frontend/src/app/pages/desktop.courseDetails.jsx [hunk 6]
```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+
+import './styles/desktop.courseDetails.scss';
+
+import CourseDetails from '../course/courseDetails';
+import CourseHeader from '../course/courseHeader';
+
+const CourseDetailsPage = (): React.Node => {
```
Comment (teodorvoicencu) [inline frontend/src/app/pages/desktop.courseDetails.jsx]: The component should match the file name, in your case either rename the component as CourseDetails or rename the file into desktop.CourseDetailsPage

File: frontend/src/app/routes/navRoutes.jsx [hunk 7]
```diff
@@ -12,6 +12,10 @@ const NavRoutes = {
         path: '/universities',
         name: 'Universities',
     },
+    course: {
+        path: '/course',
+        name: 'Course',
+    },
```
Comment (teodorvoicencu) [inline frontend/src/app/routes/navRoutes.jsx]: Remove these from the navbar please

File: frontend/src/app/course/courseHeaderExtra.jsx [hunk 8]
```diff
@@ -0,0 +1,28 @@
+// @flow
+import Person from 'react-ionicons/lib/MdPerson';
+import Albums from 'react-ionicons/lib/MdAlbums';
+import { Card } from 'react-bootstrap';
+import * as React from 'react';
+
+import './styles/courseHeaderExtra.scss';
+
+const CourseHeaderExtra = (): React.Node => {
```
Comment (teodorvoicencu) [inline frontend/src/app/course/courseHeaderExtra.jsx]: Maybe name this as CourseHeaderStats

File: frontend/src/app/course/styles/courseDetails.scss (hunk 4 above)
Comment (teodorvoicencu) [reply frontend/src/app/course/styles/courseDetails.scss]: you can write this as `padding: $ternary-padding`

Comment (teodorvoicencu) [reply frontend/src/app/course/styles/courseDetails.scss]: and change everywhere you do this

**Review by umihnea — APPROVED** [review]
_\*pops\* Nice._

**Pull request merged**